<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WhiteBoard Demo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="documentviewer.qrc">
    <normaloff>:/demos/documentviewer/images/qt-logo.png</normaloff>:/demos/documentviewer/images/qt-logo.png</iconset>
   </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <!-- WhiteBoard -->
    <item>
     <widget class="WhiteBoard" name="whiteboard"/>
    </item>
   </layout>
  </widget>
  <!-- Barra de menÃº -->
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <!-- Barra de herramientas con QPushButton -->
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <widget class="QPushButton" name="btnOpen">
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUndo">
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRedo">
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlue">
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRed">
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGreen">
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
  </widget>
  <!-- Barra de estado -->
  <widget class="QStatusBar" name="statusbar"/>
  <!-- Acciones -->
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="documentviewer.qrc"/>
 </resources>
</ui>